tests/cpp/test_combine_mul_sum.cpp

```diff
@@ -54,6 +60,8 @@ class CombineMulSumAsMmaTest : public NVFuserTest {
     }
     NVFuserTest::SetUp();
   }
+  std::unique_ptr<DisableOptionsGuard> disable_options_ptr =
+      std::make_unique<DisableOptionsGuard>();
```

Comment:
Similar to https://github.com/NVIDIA/Fuser/pull/1940, we don't need a unique ptr. 

```suggestion
 private:
  DisableOptionsGuard opt_guard_;
```

Reply:
The same comment goes to other uses of DisableOptionsGuard in this PR.